### ISSUE TITLE
Missing srp dependency

### ISSUE THREAD

**Issue by cyakimov** [maybeinit/airplay-rs#1]
Hey, I'm currently working on a PoC to make an open-source airplay client with Rust and I found your code incredibly useful to get the project started. Do you mind sharing the modified SRP library you're using here? I'll credit you in my repo once I publish the code.

**Comment by arilence** [comment]
Hey Carlos, I'm glad to hear this code helped you out. 

Keep in mind that this code only works for Apple TV Gen 3 and below. Later models changed the authentication protocol, which is one of the reasons I'm not interested in continuing this codebase at this time :joy: 

[Pyatv](https://pyatv.dev/documentation/protocols/#authentication-1) seems to have much more documented and may be of use to you.

To answer your question, I used the rust crate `srp` as a starting point.

One of the resources I referenced in my readme, [Pairing & Authenticating](https://htmlpreview.github.io/?https://github.com/philippe44/RAOP-Player/blob/master/doc/auth_protocol.html), shows the difference between the standard SRP and Apple's non-compliant implementation.

The key thing to note is in Step 2. While standard SRP calculates `k` as: 
```K = H(S)```
Apple decided to calculate `k` as: 
```K = H(S | \x00\x00\x00\x00) | H(S | \x00\x00\x00\x01)```

I don't have my exact code on hand, but you'll need to modify the [return value from calc_key()](https://github.com/RustCrypto/PAKEs/blob/9b3f1cd68dbe9b1fef0b4f9152e15be6f986856b/srp/src/client.rs#L134)
